src/tweenable.rs

```diff
@@ -791,10 +791,19 @@ impl<T> Tweenable<T> for Tween<T> {
         // If completed at least once this frame, notify the user
         if times_completed > 0 {
             if let Some(user_data) = &self.event_data {
-                events.send(TweenCompleted {
+                let event = TweenCompleted {
                     entity,
                     user_data: *user_data,
-                });
+                };
+
+                // send regular event
+                events.send(event);
+
+                // trigger all global observers
+                commands.trigger(event);
```

Comment:
This shouldn't be necessary, world-scoped observers should be able to see entity-scoped triggers too.

```suggestion
```